### DISCUSSION TITLE
Asking about ASTD-balanced-not-linked.csv file

### DISCUSSION THREAD

**Discussion by k-3bad** [aub-mind/arabert#187] [Q&A]
Hello,

where can I get the ASTD-balanced-not-linked.csv file from, as it shows the following error when running 

![image](https://user-images.githubusercontent.com/45392118/205281174-b2229e27-636c-40ce-a84e-9a57e4df5c4f.png)

**Comment by WissamAntoun** [comment]
If i remember correctly we used the same splits provided by Dahou et. al. on their github, which we sourced from this repo https://github.com/mahmoudnabil/ASTD, not sure if Dahou's repo is still up now
